CONTRIBUTING.rst

```diff
@@ -61,59 +62,70 @@ A good pull request:
 * Includes tests cases that demonstrates the previous flaw that now passes with the included patch, or demonstrates the newly added feature. Tests should have 100% code coverage.
 * Is appropriately licensed (Apache 2.0).
 
+Please keep in mind:
 
+* The benefit of contribution must be compared against the cost of maintaining the feature, as maintenance burden of new contributions are usually put on the maintainers of the project.
 
 Get Started
 -----------
 
 1. Fork the ``sceptre`` repository on GitHub.
 2. Clone your fork locally::
 
-    $ git clone git@github.org:your_name_here/sceptre.git
+    $ git clone git@github.org:<github_username>/sceptre.git
 
-3. Install your local copy into a `virtual environment <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_. Assuming you have virtualenv installed, this is how you set up your fork for local development:
+3. Install sceptre for development (we recommend you use a `virtual environment <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_):
 
-.. code-block:: shell
+   .. code-block:: shell
 
     $ cd sceptre/
-    $ # Enable your virtual environment
-    $ virtualenv env
-    $ source env/bin/activate
-    $ # Install Sceptre's development requirements
-    $ pip install -r requirements_dev.txt
-    $ # Install Sceptre's test requirements
-    $ pip install -r requirements_tests.txt
-    $ # Install sceptre locally, so you can test your changes
+    $ pip install -r requirements.txt
     $ pip install -e .
 
 4. Create a branch for local development:
 
-.. code-block:: shell
+   .. code-block:: shell
 
-    $ git checkout -b <GitHub issue number>-<short description>
+    $ git checkout -b <branch-name>
 
-5. When you're done making changes, check that your changes pass flake8 and the tests, including testing other Python versions with tox:
+5. When you're done making changes, check that your changes pass linting, unit tests and have sufficient coverage:
 
-.. code-block:: shell
+   Check linting:
 
-    $ make lint
-    $ make test-all
-    $ make coverage  # coverage should be 100%
+   .. code-block:: shell
 
-6. Make sure the changes comply with the pull request guidelines in the section on `Contributing Code`_.
+      $ make lint
+
+   Run unit tests in your current environment - (handy for quickly running unit tests):
+
+   .. code-block:: shell
+
+      $ make test
 
-7. Commit your changes:
+   Note: Sceptre aims to be compatible with Python 2 & 3, please run unit test against both versions. You will need the corresponding versions of Python installed on your system.
 
-.. code-block:: shell
+   Run unit tests using tox for Python 2.7 and 3.6:
+
+   .. code-block:: shell
+
+      $ tox -e py27
+      $ tox -e py36
```

Comment:
When running this command in my virtualenv I get the following error:


`ERROR:   py36: Error creating virtualenv. Note that some special characters (e.g. ':' and unicode symbols) in paths are not supported by virtualenv. Error details: InvocationError('Failed to get version_info for python3.6: b"pyenv: python3.6: command not found\\n\\nThe `python3.6\' command exists in these Python versions:\\n  3.6.1\\n\\n"',)`